lib/node_modules/@stdlib/math/base/special/sincospi/lib/main.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+// MODULES //
+
+var computeSincospi = require( './sincospi.js' );
+
+
+// MAIN //
+
+/**
+* Simultaneously computes the sine and cosine of a number times π.
+*
+* @param {(Array|TypedArray|Object)} [out] - destination array
+* @param {number} x - input value
+* @returns {(Array|TypedArray|Object)} sine and cosine (in radians)
+*
+* @example
+* var v = sincospi( 0.0 );
+* // returns [ ~0.0, 1.0 ]
```

Comment:
Return values need to be updated.

Reply:
`[ ~0.0, ~1.0 ]`?

Reply:
These values need to be scaled by pi, no?